src/components/ArtifactGreenwaveKaiState.tsx

```diff
@@ -152,6 +179,176 @@ export const FaceForGreenwaveKaiState: React.FC<
+                <Tab
+                    eventKey={'tab-test-known-issues'}
+                    isHidden={isTestKnownIssuesTabHidden}
+                    title={
+                        <>
+                            <TabTitleIcon>
+                                <ExclamationCircleIcon />
+                            </TabTitleIcon>
+                            <TabTitleText>Known issues</TabTitleText>
+                        </>
+                    }
+                    aria-label="Tab with known issues"
+                >
+                    <>
+                        <TestKnownIssues metadata={metadata} />
+                    </>
+                </Tab>
+                <Tab
+                    eventKey={'tab-test-deps'}
+                    isHidden={isTestDependencyTabHidden}
+                    title={
+                        <>
+                            <TabTitleIcon>
+                                <ExclamationTriangleIcon />
+                            </TabTitleIcon>
+                            <TabTitleText>Dependency</TabTitleText>
+                        </>
+                    }
+                    aria-label="Tab with dependency information"
+                >
+                    <>
+                        <TestDependency metadata={metadata} />
+                    </>
+                </Tab>
+                <Tab
+                    eventKey={'tab-test-info'}
+                    isHidden={isTestInfoTabHidden}
+                    title={
+                        <>
+                            <TabTitleIcon>
+                                <InfoCircleIcon />
+                            </TabTitleIcon>
+                            <TabTitleText>Test info</TabTitleText>
+                        </>
+                    }
+                    aria-label="Tab with test info"
+                >
+                    <>
+                        <TestInfo metadata={metadata} />
+                        <TextContent>
+                            <Text component={TextVariants.small}>
+                                CI owners can update info on metadata page.
+                            </Text>
+                        </TextContent>
+                    </>
+                </Tab>
+                <Tab
+                    eventKey={'tab-test-details'}
+                    title={
+                        <>
+                            <TabTitleIcon>
+                                <ListIcon />
+                            </TabTitleIcon>{' '}
+                            <TabTitleText>Details</TabTitleText>{' '}
+                        </>
+                    }
+                    aria-label="Tab with test details"
+                >
+                    <GreenwaveResultInfo state={state.gs} />
+                    <KaiStateMapping artifact={artifact} state={state.ks} />
+                </Tab>
+            </Tabs>
+        </>
+    );
+};
+
+export const TabWithChildren = React.forwardRef(
```

Comment:
What is the purpose of this? It's never used anywhere.

Reply:
Removed. It was leftovers from some experiments.